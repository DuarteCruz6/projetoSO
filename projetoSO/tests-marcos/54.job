WRITE [(n1,48)(g3,13)(g9,39)]
WRITE [(g1,2)(p9,49)]
READ [g3]
READ [g9]
WRITE [(i3,36)(f7,12)(c7,34)(y5,86)(h2,80)]
DELETE [g3,g9,y5,h2,g1]
WRITE [(p8,14)]
READ [p9]
WRITE [(r2,55)]
DELETE [p8,n1]
WRITE [(q3,94)(n9,49)(t1,66)(n6,91)]
READ [f7]
WRITE [(v9,100)(r0,97)(e4,77)]
BACKUP
WRITE [(y8,69)(u8,3)(a3,3)(m8,26)(j4,60)]
READ [u8]
DELETE [r0,p9,f7,u8,i3]
DELETE [j4,v9]
WRITE [(y7,13)]
WRITE [(h7,77)(w7,53)]
WRITE [(k9,37)(s8,80)(b6,23)(q1,89)(o5,82)]
WAIT 1000
WRITE [(f2,17)(z8,94)(t8,18)(k5,1)(t4,33)]
WRITE [(c4,3)(o6,10)]
WRITE [(o2,27)(n1,93)]
DELETE [c7,s8,r2,t1]
WRITE [(w6,18)]
DELETE [b6,h7]
WAIT 3000
WAIT 2000
WRITE [(x3,85)(s6,99)]
WRITE [(n3,68)(r7,2)]
WRITE [(e2,64)(h7,53)(b2,12)]
WRITE [(m2,28)(o0,13)]
SHOW
WRITE [(j2,92)]
READ [o6]
WRITE [(d0,35)]
WRITE [(f9,45)(z4,30)(b7,70)(h9,89)]
READ [k9]
READ [b2]
READ [x3]
READ [q3]
WRITE [(v9,12)(f8,60)(k7,60)(e8,80)]
READ [e2]
WAIT 2000
READ [b2]
WRITE [(q6,52)(w8,70)(b0,75)(q4,32)]
DELETE [t4,k9,f9,h7]
READ [f8]
WAIT 3000
WRITE [(o9,57)(w4,99)(g6,30)]
WRITE [(j7,11)]
WRITE [(l8,62)(y2,22)(v1,89)(m1,64)(i1,67)]
WRITE [(t0,61)(a8,71)]
DELETE [o6]
WRITE [(t6,4)(p0,37)(y3,21)(m9,56)(g2,58)]
WRITE [(h2,27)(q0,90)(g3,34)(h2,46)]
WRITE [(r4,16)(h5,90)(s4,51)(n2,59)]
WRITE [(b6,74)]
WRITE [(w3,94)(a2,82)(s7,62)]
WRITE [(o6,76)(g5,49)]
DELETE [y7,o9,z4,k5,b6]
WRITE [(e5,58)(g8,87)]
BACKUP
READ [b2]
WRITE [(p9,79)(f9,90)(l4,10)]
DELETE [o2,q4,n3,r7]
DELETE [w6,q6,g2,n2,f9]
WRITE [(u4,94)(s8,37)]
READ [l4]
WRITE [(m0,65)(c9,37)(u3,9)(l6,40)]
READ [h5]
WAIT 3000
READ [s4]
READ [m0]
READ [m2]
DELETE [e4,a2,x3,e8]
WAIT 3000
WRITE [(r1,9)(p8,24)]
WRITE [(j1,25)(a9,26)(f0,99)(e8,64)(e3,37)]
DELETE [f0,c9,q0,v1,s4]
WRITE [(d5,39)(l1,23)(z5,8)(q7,18)]
BACKUP
WRITE [(j3,97)(i9,99)]
WRITE [(z3,0)(i7,22)]
WAIT 1000
DELETE [q3,r4,l1,i7]
