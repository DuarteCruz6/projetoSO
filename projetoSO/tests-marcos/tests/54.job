SHOW
WRITE [(x7,62)(l4,81)(j9,75)(u6,41)(c7,70)]
BACKUP
DELETE [c7]
SHOW
WRITE [(k0,22)(k8,51)(t1,57)(m5,71)(b0,71)]
WRITE [(b2,35)]
READ [l4]
READ [b0]
WRITE [(z4,90)(z9,34)(l9,90)(y8,54)(j3,7)]
WRITE [(o4,37)(n7,28)]
READ [b2]
BACKUP
READ [b0]
WRITE [(e7,88)(t3,49)(c7,6)(r8,40)]
BACKUP
WRITE [(g6,72)(z6,54)]
WRITE [(k4,88)(f9,33)(i0,2)(o9,97)]
BACKUP
SHOW
READ [k0]
BACKUP
DELETE [l4,c7,b0,o4,y8]
WAIT 3000
WRITE [(h3,90)(u8,8)(m1,37)(h8,66)]
WRITE [(w3,57)(p9,68)]
WRITE [(a3,23)]
WRITE [(r3,66)(f6,5)(i8,69)(h9,27)(x0,67)]
READ [z4]
READ [m1]
WRITE [(h7,61)(c9,91)(r2,4)(e3,32)(f8,57)]
SHOW
WRITE [(x8,47)(b4,79)]
WRITE [(m3,54)(b8,71)(w1,96)(d8,86)(y6,39)]
READ [e7]
WRITE [(z5,59)]
BACKUP
WRITE [(a4,96)(m6,60)(q4,3)(w0,37)(p6,34)]
WRITE [(y1,46)]
WRITE [(s7,78)(h1,67)(o2,55)(q2,54)(w7,96)]
READ [t3]
WAIT 2000
WRITE [(n4,20)(b7,28)(t6,78)]
DELETE [u8,y1]
WRITE [(v3,61)(a0,30)]
WRITE [(y8,43)(l1,50)]
WRITE [(k1,100)]
DELETE [t3,r3]
WRITE [(l5,8)(i9,47)(o0,83)(g1,96)(k3,90)]
READ [n4]
DELETE [g1,o2,l9,w3]
BACKUP
READ [i9]
WRITE [(o7,82)(w5,69)(t3,18)(w2,67)]
READ [n4]
READ [p9]
DELETE [k8]
WRITE [(m0,2)(a5,86)(s6,94)]
SHOW
READ [j9]
WRITE [(j1,93)(b9,13)(w4,24)(l0,99)(q5,32)]
WRITE [(e8,39)]
WRITE [(t0,39)(d2,72)]
BACKUP
WRITE [(s3,53)(y7,2)(v4,13)(x9,25)]
WRITE [(d5,50)(i2,67)(k2,12)(o3,46)(z1,84)]
WRITE [(q3,27)(q7,87)(y0,65)(u5,44)]
WRITE [(t9,96)(i4,55)(s1,50)(p7,78)(p0,34)]
WRITE [(p2,15)(l8,24)(a6,52)]
DELETE [j1,q5]
DELETE [t9,f6,l0,m0]
SHOW
WRITE [(v1,52)(l6,83)(v7,49)(w3,79)]
