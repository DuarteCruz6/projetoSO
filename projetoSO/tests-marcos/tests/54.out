(j9, 75)
(l4, 81)
(u6, 41)
(x7, 62)
[(l4,81)]
[(b0,71)]
[(b2,35)]
[(b0,71)]
(b2, 35)
(b0, 71)
(c7, 6)
(e7, 88)
(f9, 33)
(g6, 72)
(i0, 2)
(j3, 7)
(j9, 75)
(k4, 88)
(k8, 51)
(k0, 22)
(l9, 90)
(l4, 81)
(m5, 71)
(n7, 28)
(o9, 97)
(o4, 37)
(r8, 40)
(t3, 49)
(t1, 57)
(u6, 41)
(x7, 62)
(y8, 54)
(z6, 54)
(z9, 34)
(z4, 90)
[(k0,22)]
[(z4,90)]
[(m1,37)]
(a3, 23)
(b2, 35)
(c9, 91)
(e3, 32)
(e7, 88)
(f8, 57)
(f6, 5)
(f9, 33)
(g6, 72)
(h7, 61)
(h9, 27)
(h8, 66)
(h3, 90)
(i8, 69)
(i0, 2)
(j3, 7)
(j9, 75)
(k4, 88)
(k8, 51)
(k0, 22)
(l9, 90)
(m1, 37)
(m5, 71)
(n7, 28)
(o9, 97)
(p9, 68)
(r2, 4)
(r3, 66)
(r8, 40)
(t3, 49)
(t1, 57)
(u8, 8)
(u6, 41)
(w3, 57)
(x0, 67)
(x7, 62)
(z6, 54)
(z9, 34)
(z4, 90)
[(e7,88)]
[(t3,49)]
[(n4,20)]
[(i9,47)]
[(n4,20)]
[(p9,68)]
(a5, 86)
(a0, 30)
(a4, 96)
(a3, 23)
(b7, 28)
(b8, 71)
(b4, 79)
(b2, 35)
(c9, 91)
(d8, 86)
(e3, 32)
(e7, 88)
(f8, 57)
(f6, 5)
(f9, 33)
(g6, 72)
(h1, 67)
(h7, 61)
(h9, 27)
(h8, 66)
(h3, 90)
(i9, 47)
(i8, 69)
(i0, 2)
(j3, 7)
(j9, 75)
(k3, 90)
(k1, 100)
(k4, 88)
(k0, 22)
(l5, 8)
(l1, 50)
(m0, 2)
(m6, 60)
(m3, 54)
(m1, 37)
(m5, 71)
(n4, 20)
(n7, 28)
(o7, 82)
(o0, 83)
(o9, 97)
(p6, 34)
(p9, 68)
(q2, 54)
(q4, 3)
(r2, 4)
(r8, 40)
(s6, 94)
(s7, 78)
(t3, 18)
(t6, 78)
(t1, 57)
(u6, 41)
(v3, 61)
(w2, 67)
(w5, 69)
(w7, 96)
(w0, 37)
(w1, 96)
(x8, 47)
(x0, 67)
(x7, 62)
(y8, 43)
(y6, 39)
(z5, 59)
(z6, 54)
(z9, 34)
(z4, 90)
[(j9,75)]
(a6, 52)
(a5, 86)
(a0, 30)
(a4, 96)
(a3, 23)
(b9, 13)
(b7, 28)
(b8, 71)
(b4, 79)
(b2, 35)
(c9, 91)
(d5, 50)
(d2, 72)
(d8, 86)
(e8, 39)
(e3, 32)
(e7, 88)
(f8, 57)
(f9, 33)
(g6, 72)
(h1, 67)
(h7, 61)
(h9, 27)
(h8, 66)
(h3, 90)
(i4, 55)
(i2, 67)
(i9, 47)
(i8, 69)
(i0, 2)
(j3, 7)
(j9, 75)
(k2, 12)
(k3, 90)
(k1, 100)
(k4, 88)
(k0, 22)
(l8, 24)
(l5, 8)
(l1, 50)
(m6, 60)
(m3, 54)
(m1, 37)
(m5, 71)
(n4, 20)
(n7, 28)
(o3, 46)
(o7, 82)
(o0, 83)
(o9, 97)
(p2, 15)
(p0, 34)
(p7, 78)
(p6, 34)
(p9, 68)
(q7, 87)
(q3, 27)
(q2, 54)
(q4, 3)
(r2, 4)
(r8, 40)
(s1, 50)
(s3, 53)
(s6, 94)
(s7, 78)
(t0, 39)
(t3, 18)
(t6, 78)
(t1, 57)
(u5, 44)
(u6, 41)
(v4, 13)
(v3, 61)
(w4, 24)
(w2, 67)
(w5, 69)
(w7, 96)
(w0, 37)
(w1, 96)
(x9, 25)
(x8, 47)
(x0, 67)
(x7, 62)
(y0, 65)
(y7, 2)
(y8, 43)
(y6, 39)
(z1, 84)
(z5, 59)
(z6, 54)
(z9, 34)
(z4, 90)
